clients/src/main/java/org/apache/kafka/common/utils/Utils.java

```diff
@@ -351,7 +351,7 @@ public static String formatAddress(String host, Integer port) {
     }
 
     public static <K, V> String mkString(Map<K, V> map) {
```

Comment:
Seems like we should just remove this method if it's the same as `Map.toString()`.